packages/components/ScreenContainer/ScreenTitle.tsx

```diff
@@ -0,0 +1,15 @@
+import { ReactNode } from "react";
+
+import { BrandText } from "../BrandText";
+
+import { fontRegular15 } from "@/utils/style/fonts";
+
+export const ScreenTitle: React.FC<{
```

Comment:
According to all the changes I see in this PR, the children is always text and in no case children can be undefined. So is that OK to change children to mandatory props and children become string instead of ReactNode ?